_docs/install/cloud/azure.md

```diff
@@ -5,4 +5,204 @@ anchor: install
 module: install/cloud/azure
 ---
 
-{% include wip.md %}
+# Azure
+
+## Azure Container Service (AKS) Installation
+
+Since AKS runs `kubelet` in a container, an additional parameter was
+added to the StorageOS driver in Kubernetes 1.10 to allow configuring the
+location where StorageOS presents volume devices to `kubelet` to see the
+devices.
+
+We expect StorageOS to run on AKS when it supports Kubernetes
+1.10.  Until then, you may deploy your own Kubernetes cluster on Azure using the acs-engine instructions below.
+
+## acs-engine
+
+`acs-engine` is an open-source tool from Microsoft that uses to deploy AKS clusters, and it can be used to deploy your own Kubernetes cluster on Azure.  It allows
+you to specify the version of Kubernetes and specific features.
+
+For more information and installation instructions, consult the
+[acs-engine kubernetes guide](https://github.com/Azure/acs-engine/blob/master/docs/kubernetes/deploy.md)
+
+There are various [examples of kubernetes cluster definitions](https://github.com/Azure/acs-engine/tree/master/examples) for acs-engine, but the important thing is to specify kubernetes 1.10 as StorageOS requires this version to support running with kubelet in a container.
+
+Example definition (saved as `kubernetes.json` for use in the instructions below):
+
+```json
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "orchestratorRelease": "1.10"
+    },
+    "masterProfile": {
+      "count": 1,
+      "dnsPrefix": "",
+      "vmSize": "Standard_D2_v2"
+    },
+    "agentPoolProfiles": [
+      {
+        "name": "agentpool1",
+        "count": 3,
+        "vmSize": "Standard_D2_v2",
+        "availabilityProfile": "AvailabilitySet"
+      }
+    ],
+    "linuxProfile": {
+      "adminUsername": "azureuser",
+      "ssh": {
+        "publicKeys": [
+          {
+            "keyData": ""
+          }
+        ]
+      }
+    },
+    "servicePrincipalProfile": {
+      "clientId": "",
+      "secret": ""
+    }
+  }
+}
+```
+
+To deploy a cluster through acs-engine you need your Azure Subscription ID. 
+The [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) is a great way to get the ID of your subscription. If you don't have the Azure CLI installed then you can sign in to [Azure Cloud Shell](https://shell.azure.com) to run the commands there:
```

Comment:
"The [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) is a great way to get the ID of your subscription."
=>
"You can retrieve your subscription ID through the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)."